### PR TITLE
Attributes - Add support for FIREWEAPON AI ability

### PR DESCRIPTION
**Pull request by PlayerBotPro**
**When merged this pull request will:**
- Title.
- Documentation: https://community.bistudio.com/wiki/disableAI

### CODE REVIEW

File: addons/ai/stringtable.xml [hunk 1]
```diff
@@ -404,5 +404,17 @@
             <Chinese>武器瞄準</Chinese>
             <Italian>Obiettivo dell'arma</Italian>
         </Key>
+        <Key ID="STR_ZEN_AI_FireWeapon">
+            <English>Weapon Fire</English>
```
Comment (mharis001) [inline addons/ai/stringtable.xml:408]: Why is this backwards? Should be "Fire Weapon".

File: addons/ai/stringtable.xml [hunk 2]
```diff
@@ -404,5 +404,17 @@
...
+            <French>Tir d'arme</French>
+            <German>Waffenfeuer</German>
+            <Polish>Ogień z broni</Polish>
+            <Korean>무기 발사</Korean>
+            <Japanese>武器発射</Japanese>
+            <Spanish>Disparo de arma</Spanish>
+            <Chinesesimp>武器开火</Chinesesimp>
+            <Chinese>武器開火</Chinese>
+            <Italian>Fuoco dell'arma</Italian>
```
Comment (mharis001) [inline addons/ai/stringtable.xml:417]: Where are these translations from?

File: addons/attributes/initAttributes.inc.sqf [hunk 3]
```diff
@@ -946,7 +946,8 @@
         [6.5,  3, 6.5, ELSTRING(ai,Suppression)],
         [13,   3, 6.5, ELSTRING(ai,Target)],
         [19.5, 3, 6.5, ELSTRING(ai,TeamSwitch)],
-        [0,    4, 6.5, ELSTRING(ai,WeaponAim)]
+        [0,    4, 6.5, ELSTRING(ai,WeaponAim)],
+        [6.5,  4, 6.5, ELSTRING(ai,FireWeapon)]
```
Comment (mharis001) [inline addons/attributes/initAttributes.inc.sqf:950]: Should be sorted alphabetically (after changing to "Fire Weapon").

**Review by mharis001 — CHANGES_REQUESTED** [review]

File: addons/ai/stringtable.xml (hunk 1 above)
Comment (Kexanone) [reply addons/ai/stringtable.xml:408]: If it prevents AI from firing their weapon, maybe just call it shooting.

File: addons/ai/stringtable.xml (hunk 2 above)
Comment (PlayerBotPro) [reply addons/ai/stringtable.xml]: ChatGPT. Removed in new commit cuz I dont trust it

File: addons/ai/stringtable.xml (hunk 1 above)
Comment (PlayerBotPro) [reply addons/ai/stringtable.xml]: Changed to "Fire Weapon" and sorted alphabetically

File: addons/attributes/initAttributes.inc.sqf (hunk 3 above)
Comment (PlayerBotPro) [reply addons/attributes/initAttributes.inc.sqf]: Sorted alphabetically.

File: addons/ai/stringtable.xml (hunk 2 above)
Comment (PlayerBotPro) [reply addons/ai/stringtable.xml]: Im sorry I didnt know that. Thank you for helping me improve.

**Review by mharis001 — APPROVED** [review]

**Review by mharis001 — APPROVED** [review]

**Pull request merged**